### PR TITLE
Sorting Algorithm Program with Timings

### PR DESCRIPTION
**Pull request by souvikpal2000**
My Program contains following Sorting Algorithm : 

1. Bubble Sort
2. Selection Sort
3. Insertion Sort
4. Quick Sort

My Program will show the timings of all the sorting algorithm to sort random data.

image

### CODE REVIEW

File: Sorting_Algo_withTimings/ArrayRoutines.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+     * Displays the values stored in an array of ints
+     * @param a  the array
+     */
+    public static void display(int[] a) {
+        for (int i = 0; i < a.length; i++) {
+            System.out.println("element " + i + ": " + a[i]); //<-- the formatting is up to you
+        }
+    }
+    
+    //Other methods from ArrayRoutines not copied across
```
Comment (rohan472000) [inline Sorting_Algo_withTimings/ArrayRoutines.java:153]:
```suggestion
    //Other methods from ArrayRoutines not copied across
```

What does it mean??

Comment (rohan472000) [inline Sorting_Algo_withTimings/ArrayRoutines.java:153]: What does it mean ?

Comment (souvikpal2000) [reply Sorting_Algo_withTimings/ArrayRoutines.java:153]: Everything is mentioned in comments.

File: Sorting_Algo_withTimings/SortTiming.java [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        bubbleTime = timeSort(bubbleSorted, Alg.BUBBLE);
+        selectTime = timeSort(selectSorted, Alg.SELECT);
+        insertTime = timeSort(insertSorted, Alg.INSERT);
+        quickTime = timeSort(quickSorted, Alg.QUICK);
+        System.out.println("done");
+
+        if (displayAfter) {
+            System.out.println("Original data");
+            ArrayRoutines.display(data);
+            //Use own display so can present them simultaneously
```
Comment (rohan472000) [inline Sorting_Algo_withTimings/SortTiming.java:82]:
```suggestion
           
```

File: Sorting_Algo_withTimings/SortTiming.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
+import java.util.Arrays;
+import java.util.Random;
+
+/**
+ * Utility for comparing the runtime of various sorting algorithms.
+ * Part of 6.3DN Sorting Algorithms.
+ * 
+ * @author Ibrahim Hossain Limon
+ * @version 23 Aug 2023
```
Comment (rohan472000) [inline Sorting_Algo_withTimings/SortTiming.java:9]: Explain these lines...6.3DN, author, version??

File: Sorting_Algo_withTimings/ArrayRoutines.java (hunk 1 above)
Comment (rohan472000) [reply Sorting_Algo_withTimings/ArrayRoutines.java:153]: I'm asking about line 153

File: Sorting_Algo_withTimings/SortTiming.java (hunk 3 above)
Comment (souvikpal2000) [reply Sorting_Algo_withTimings/SortTiming.java:9]: > Explain these lines...6.3DN, author, version??

This is actually code is written by somebody else. I have only included the timings functionality. If you want you can remove this lines.

File: Sorting_Algo_withTimings/SortTiming.java (hunk 2 above)
Comment (souvikpal2000) [reply Sorting_Algo_withTimings/SortTiming.java:82]: This was pre defined template. That's why comments left as it was. You can delete those

File: Sorting_Algo_withTimings/SortTiming.java (hunk 3 above)
Comment (rohan472000) [reply Sorting_Algo_withTimings/SortTiming.java:9]: Somebody else!!!! Why not you?

You can write those codes on your own, kindly don't paste somebody else's code here as I don't know about their licence on code.

**Comment by rohan472000** [comment]
Closing due to no activity.

**Pull request closed without merging**